src/EditorFeatures/TestUtilities/FakeLoggingServiceInternal.cs

```diff
@@ -0,0 +1,52 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Composition;
+using Microsoft.VisualStudio.Composition;
+using Microsoft.VisualStudio.Text.Utilities;
+
+namespace Microsoft.CodeAnalysis.Editor.UnitTests
+{
+    // Provide an export of ILoggingServiceInternal to work around https://devdiv.visualstudio.com/DevDiv/_workitems/edit/570290
```

Comment:
This bug has already been fixed, if we are unable to delete this now we need to refile a new bug.